packages/builder/src/components/portal/licensing/LicensingOverlays.svelte

```diff
@@ -82,31 +81,25 @@
     }
   }
 
+  $: if (!userLoaded && $auth.user) {
+    userLoaded = true
+  }
+
   $: if (
     userLoaded &&
-    licensingLoaded &&
-    loaded &&
-    isEnabled(FEATURE_FLAGS.LICENSING)
+    $licensing.usageMetrics &&
+    domLoaded &&
+    !licensingLoaded
   ) {
+    licensingLoaded = true
     queuedModals = processModals()
     queuedBanners = getBanners()
     showNextModal()
     banner.queue(queuedBanners)
   }
 
   onMount(async () => {
-    auth.subscribe(state => {
-      if (state.user && !userLoaded) {
-        userLoaded = true
-      }
-    })
-
-    licensing.subscribe(state => {
-      if (state.usageMetrics && !licensingLoaded) {
-        licensingLoaded = true
-      }
-    })
-    loaded = true
+    domLoaded = true
```

Comment:
Out of curiosity how come we need the additional `domLoaded` flag?

Reply:
I removed flag initially and then, when attempting to show the licensing modals, it would periodically fail because they hadn't fully initialised. I changed the name to domLoaded just for clarity.